app/src/main/java/uk/gov/onelogin/tokens/usecases/SaveToSecureStore.kt

```diff
@@ -34,14 +34,12 @@ class SaveToSecureStoreImpl @Inject constructor(
     override suspend fun invoke(context: FragmentActivity, key: String, value: String) {
         biometricPreferenceHandler.getBioPref()?.let { bioPref ->
             if (bioPref == BiometricPreference.NONE) return
-
-            val secureStore: SecureStore = SharedPrefsStore(
-                context,
-                configuration = SecureStorageConfiguration(
-                    Keys.SECURE_STORE_ID,
-                    Mapper.mapAccessControlLevel(bioPref)
-                )
+            val configuration = SecureStorageConfiguration(
+                Keys.SECURE_STORE_ID,
+                Mapper.mapAccessControlLevel(bioPref)
             )
+            val secureStore: SecureStore = SharedPrefsStore()
```

Comment:
Same as above, maybe should be injecting the SecureStore now

Reply:
done 0f6151a42318989a835ed8b94724b1e65db104a1